brain/browser_use_adapter.py

```
@@ -576,10 +663,21 @@ async def close_session(self, session_id: str) -> None:
     async def _close_browser(self) -> None:
         self._stop_overlay()
         if self._browser_session is not None:
+            # Grab browser PID before stopping so we can force-kill if .stop() hangs
+            watchdog = getattr(self._browser_session, "_local_browser_watchdog", None)
+            proc = getattr(watchdog, "_subprocess", None) if watchdog else None
+            browser_pid = None
+            if proc is not None:
+                try:
+                    browser_pid = proc.pid
```

Ruff (0.15.2)

[error] Local variable `browser_pid` is assigned to but never used

Remove assignment to unused variable `browser_pid`

(F841)